src/main/scala/com/fulcrumgenomics/alignment/Alignment.scala

```diff
@@ -351,8 +351,8 @@ case class Alignment(query: Array[Byte],
         // Don't include, beyond the range we're interested
       }
       else if (currStart >= start && currEnd <= end) {
-        // Contained within the target region; ignore insertions that are at the ends of the desired region
-        if (elementConsumes || (currStart != start && currStart != end)) {
+        // Contained within the target region; ignore insertions at the boundaries
+        if (elementConsumes || currStart != start) {
```

Comment:
"ignore insertions at the boundaries".  insertions relative to what?  How about: "ignore gaps at the boundaries that do not consume bases in the sequence"